SurfaceTopography/HeightContainer.py

```diff
@@ -63,7 +63,8 @@ class Error(Exception):
         # pylint: disable=missing-docstring
         pass
 
-    def __init__(self, info={}, communicator=MPI.COMM_WORLD):
+    def __init__(self, unit=None, info={}, communicator=MPI.COMM_WORLD):
+        self._unit = unit
```

Comment:
Two Ideas: 

- We could show a deprecation warning here if `info` still has a unit attribute. Would uncover if some of our tests still rely on `info`
- We should raise an error if `info` and `unit` and it's not equal to the given `unit`. This would uncover if some tests weren't correctly initialized.

Old code would not be affected, since both `unit` and `info['unit']` need to be given for a warning/error.

Reply:
This is a good idea. I implemented this. It will raise an error if the parameter is passed in the info dictionary.